client/src/app/submission/metadata-list/metadata-list.component.ts

```diff
@@ -25,7 +25,6 @@ export class MetadataListComponent implements OnInit, OnDestroy {
 
   @Input() title: string;
   @Input() metadataList;
-  @Input() metadataType;
```

Comment:
Linking issue found where metadataType is still referenced in the HTML:
https://github.com/ebi-ait/ingest-ui/blob/4a2d37c356388510159eeb4249cf6c3b0ac9a213/src/app/submission/components/metadata-list/metadata-list.component.html#L158